include/highfive/bits/H5Utils.hpp

```diff
@@ -109,6 +145,14 @@ struct manipulator<Eigen::Matrix<T, M, N>> {
 
     static const size_t n_dims = 2;
     static const size_t r_n_dims = n_dims + manipulator<value_type>::r_n_dims;
+
+    static std::vector<size_t> get_sizes(const type& val) {
```

Comment:
cant we do
`std::array<size_t, n_dims>` ? (maybe ndims must be constexpr

Reply:
ok, for `dimensions`,
I can return the `std::array`

Reply:
for the `std::array` we agree that it will not be the `n_dims` but the `r_n_dims`, right?